### ISSUE TITLE
Thinks that BJ is global event

### ISSUE THREAD

**Issue by at808** [dankgrinder/dankgrinder#20]
**Expected behavior**
The bot would automatically respond to global events alone.

**Actual behavior**
Whenever anyone does the blackjack command, it responds to it as if it were a global event

**Steps to reproduce**
Activate the bot, then do the blackjack command in the same channel that the bot is using

**Logs**
```
{"delay":"389ms","level":"info","msg":"sending command: pls hl","time":"2021-03-19T17:22:40-10:00","typing":"850ms"}
{"delay":"493ms","level":"info","msg":"responding to search: uber","time":"2021-03-19T17:22:46-10:00","typing":"769ms"}
{"delay":"435ms","level":"info","msg":"sending command: pls beg","time":"2021-03-19T17:22:47-10:00","typing":"989ms"}
{"delay":"286ms","level":"info","msg":"sending command: pls pm","time":"2021-03-19T17:22:48-10:00","typing":"981ms"}
{"delay":"202ms","level":"info","msg":"sending command: pls fish","time":"2021-03-19T17:22:54-10:00","typing":"1.07s"}
{"delay":"298ms","level":"info","msg":"responding to fishing or hunting event: get the camera ready mom","time":"2021-03-19T17:22:58-10:00","typing":"3.246s"}

{"delay":"275ms","level":"info","msg":"responding to global event: h` to **hit**, type `s` to **stand**, or type `e","time":"2021-03-19T17:23:01-10:00","typing":"6.489s"}

{"delay":"363ms","level":"info","msg":"responding to postmeme: c","time":"2021-03-19T17:23:08-10:00","typing":"328ms"}
{"delay":"262ms","level":"info","msg":"sending command: pls hunt","time":"2021-03-19T17:23:10-10:00","typing":"1.306s"}

{"delay":"408ms","level":"info","msg":"responding to global event: h` to **hit**, type `s` to **stand**, or type `e","time":"2021-03-19T17:23:16-10:00","typing":"6.486s"}

{"delay":"214ms","level":"info","msg":"responding to global event: h` to **hit**, type `s` to **stand**, or type `e","time":"2021-03-19T17:23:23-10:00","typing":"6.531s"}

{"delay":"184ms","level":"info","msg":"sending command: pls search","time":"2021-03-19T17:23:31-10:00","typing":"1.535s"}
{"delay":"173ms","level":"info","msg":"responding to search: pantry","time":"2021-03-19T17:23:33-10:00","typing":"898ms"}

{"delay":"130ms","level":"info","msg":"responding to global event: h` to **hit**, type `s` to **stand**, or type `e","time":"2021-03-19T17:23:35-10:00","typing":"6.576s"}

{"delay":"155ms","level":"info","msg":"sending command: pls hl","time":"2021-03-19T17:23:42-10:00","typing":"1.005s"}
{"delay":"278ms","level":"info","msg":"responding to highlow: low","time":"2021-03-19T17:23:44-10:00","typing":"469ms"}
{"delay":"455ms","level":"info","msg":"sending command: pls beg","time":"2021-03-19T17:23:45-10:00","typing":"1.033s"}
{"delay":"344ms","level":"info","msg":"sending command: pls pm","time":"2021-03-19T17:23:46-10:00","typing":"917ms"}
{"delay":"132ms","level":"info","msg":"responding to postmeme: r","time":"2021-03-19T17:23:49-10:00","typing":"350ms"}
{"delay":"187ms","level":"info","msg":"sending command: pls fish","time":"2021-03-19T17:23:49-10:00","typing":"1.139s"}
```

**Comment by dankgrinder** [comment]
Thanks for the report, I'll look into this

**Comment by dankgrinder** [comment]
Should be fixed in [0.10.0](https://github.com/dankgrinder/dankgrinder/releases/tag/0.10.0)
